keepassxc-browser/options/options.html

```diff
@@ -543,7 +543,7 @@ <h2 class="pb-3 mt-0" data-i18n="optionsConnectedDatabasesTab"></h2>
               <div class="card my-4 shadow">
                 <div class="card-body">
                   <div class="table-responsive">
-                    <table class="table table-sm table-striped table-bordered">
+                    <table class="table table-sm table-striped">
```

Comment:
Why is `table-bordered` removed from every table? After the change there are no borders in left or right.

Reply:
I think it gives the pages a cleaner look, but if you feel strongly about it then I can revert this change.

Reply:
I understand your point. Maybe bigger design changes should be done in a separate PR anyway.